api/api-doc/storage_service.json

```diff
@@ -1756,6 +1756,22 @@
                      "allowMultiple":false,
                      "type":"string",
                      "paramType":"query"
+                  },
+                  {
+                     "name":"load_and_stream",
```

Comment:
Should we make it the default? Or even the only option? I don't see why the existing option is useful.

Reply:
I think it is better to leave the default as it is now, not to surprise user. We do have user and procedures to use the old nodetool refresh.

Reply:
Hmm. We could have a new nodetool command, `nodetool import`.

Perhaps we can even have a REST API that accepts an sstable as input (and pushes it through HTTP POST). We can then issue `nodetool upload /path/to/sstable` and get the sstable streamed through http and then through this mechanism.

Reply:
This sounds neat. However, to upload TB of sstables in production, I would rather use rsync instead of HTTP POST.

Reply:
POST should be able to stream, and it has the advantage of using our I/O scheduler to write, so we don't overwhelm the disk.

Reply:
User can set a bandwidth limitation to rsync to limit the impact. What kind of tool the user use to POST + stream on the sender side?

Reply:
curl, or an updated nodetool